include/zmq.h

```diff
@@ -381,6 +381,7 @@ ZMQ_EXPORT const char *zmq_msg_gets (const zmq_msg_t *msg_,
 #define ZMQ_GSSAPI_PRINCIPAL_NAMETYPE 90
 #define ZMQ_GSSAPI_SERVICE_PRINCIPAL_NAMETYPE 91
 #define ZMQ_BINDTODEVICE 92
+#define ZMQ_RECONNECT_STOP 109
```

Comment:
this needs to be in the DRAFT section below, and it needs to be duplicated in src/zmq_draft.hpp